pkg/controller/jobframework/webhook/defaulter.go

```diff
@@ -58,3 +70,32 @@ func (h *losslessDefaulter) Handle(ctx context.Context, req admission.Request) a
 	}
 	return response
 }
+
+// getUnknownPaths returns a list of unknown paths.
+func (h *losslessDefaulter) getUnknownPaths(raw []byte) (sets.Set[string], error) {
+	obj := h.object.DeepCopyObject()
+	strictErrors, err := json.UnmarshalStrict(raw, obj, json.DisallowUnknownFields)
+	if err != nil {
+		return nil, err
+	}
+	paths := sets.New[string]()
+	for _, err := range strictErrors {
+		paths.Insert(err.(json.FieldError).FieldPath())
+	}
+	return paths, nil
+}
+
+// convertToFieldErrorPath converts the given path into the FieldError path.
+func convertToFieldErrorPath(path string) string {
+	// Replace /foo/bar to foo/bar
+	path, _ = strings.CutPrefix(path, "/")
```

Comment:
Maybe use `TrimLeft` so that you only have one value returned.

Reply:
Done